src/miner.cpp

```diff
@@ -35,6 +35,7 @@
 #include <llmq/utils.h>
 #include <masternode/payments.h>
 #include <spork.h>
+#include <validation.h>
```

Comment:
for some reason something in 42ea443e7178161532e69f4534156b8fdfe1359b feels wrong